rskj-core/src/main/java/org/ethereum/core/genesis/BlockChainLoader.java

```diff
@@ -69,19 +69,20 @@ public void loadBlockchain() {
 
             BigInteger initialNonce = config.getBlockchainConfig().getCommonConstants().getInitialNonce();
             Genesis genesis = GenesisLoader.loadGenesis(config, config.genesisInfo(), initialNonce, true);
-            for (ByteArrayWrapper address : genesis.getPremine().keySet()) {
-                RskAddress addr = new RskAddress(address.getData());
-                repository.createAccount(addr);
+            for (RskAddress address : genesis.getPremine().keySet()) {
```

Comment:
I'd keep `addr` name for consistency.

Reply:
@ajlopezrsk please use 'addr', it's the name we're using everywhere else